deps.diff

```diff
@@ -1,9 +0,0 @@
-diff --no-dereference -N -r current/vendor/github.com/containernetworking/plugins/plugins/ipam/host-local/backend/disk/backend.go updated/vendor/github.com/containernetworking/plugins/plugins/ipam/host-local/backend/disk/backend.go
-59c59
-< 	f, err := os.OpenFile(fname, os.O_RDWR|os.O_EXCL|os.O_CREATE, 0600)
----
-> 	f, err := os.OpenFile(fname, os.O_RDWR|os.O_EXCL|os.O_CREATE, 0644)
-77c77
-< 	err = ioutil.WriteFile(ipfile, []byte(ip.String()), 0600)
----
-> 	err = ioutil.WriteFile(ipfile, []byte(ip.String()), 0644)
```

Comment:
This file got deleted; it's needed to preserve a local patch for a CVE. I think if you use `make update-deps` rather than `go mod vendor` it will do the right magic automatically. (Do that after un-deleting this file.)